maintainers/team-list.nix

```diff
@@ -364,6 +364,17 @@ with lib.maintainers; {
     shortName = "Kodi";
   };
 
+  libretro = {
+    members = [
+      aanderse
```

Comment:
@aanderse I add you to this team since you're always helping with reviews. If you don't want it just ask me and I can remove your name.

Reply:
Pinging @edwtjo and @MP2E too, I added you both to this team since you're already maintainers of both the RetroArch and libretro packages. If you don't want to be here just give me a call.